UnitySetup/UnitySetup.psm1

```diff
@@ -1179,6 +1179,8 @@ function Uninstall-UnitySetupInstance {
    Get the Unity versions installed and their locations
 .PARAMETER BasePath
    Under what base patterns should we look for Unity installs?
+   Defaults to Unity default locations by platform
+   Default can be configured by comma separated paths in $env:UNITY_SETUP_INSTANCE_DEFAULT_BASEPATH
```

Comment:
This is kind of confusing since we refer to this as a "pattern" as well as a "path".

Reply:
I agree, think the arg should be 'Pattern', but don't want to break the api. Out of scope of this change I think too.